db/migrate/20160223172116_rolify_create_roles.rb

```diff
@@ -0,0 +1,19 @@
+class RolifyCreateRoles < ActiveRecord::Migration
+  def change
+    create_table(:roles) do |t|
+      t.string :name
```

Comment:
Good point. I didn't realize it was autogenerated.

On Thu, Apr 7, 2016 at 12:47 PM, Melody <notifications@github.com> wrote:

> In db/migrate/20160223172116_rolify_create_roles.rb
> <https://github.com/sassafrastech/madeline_system/pull/63#discussion_r58905040>
> :
>
> > @@ -0,0 +1,19 @@
> > +class RolifyCreateRoles < ActiveRecord::Migration
> > +  def change
> > +    create_table(:roles) do |t|
> > +      t.string :name
>
> I don't usually like to modify migrations generated by gems, for all I
> know some part of their behavior is reliant on these constraints not
> existing at the database level. I can attempt to add them in another
> migration and see if anything breaks, but I didn't want to make any
> assumptions about the implementation here.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/sassafrastech/madeline_system/pull/63/files/533ddfacb0a278b1adc7f9cc6abbc0bcac2d1ae5#r58905040>
>



-- 
Tom Smyth

Worker-Owner, Sassafras Tech Collective
Specializing in innovative, usable tech for social change
sassafras.coop *·* @sassafrastech

Resident, Touchstone Cohousing
touchstonecohousing.org